index.html

```diff
@@ -22,25 +23,195 @@
+                    <h2 class="title">Tonic</h2>
+                    <ul class="Can-2015">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="back">Back End Dev</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p class="info">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="lang">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+
+            <!-- ******Project 4******  -->
+            <div class="Project">
+                <img class="pic1" src="./Images/snap4.svg" alt="First-Project">
+                <div class="body">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <ul class="Can-2015">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="back">Back End Dev</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p class="info">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="lang">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+
+        </section>
+        <!-- /*About myself ----------------------------section*/ -->
+        <section class="myself">
+            <div class="Top-part">
+                <h3>About Myself</h3>
+                <p class="self">
+                    Hello I'm a software developer! I can help
+                    you build a product, feature or website.
+                    Look through some of my work and
+                    experience! if you like what you see and
+                    have a project you need coded, don't
+                    hesitate to contact me.
+                </p>
+                <p class="connect">LET'S CONNECT!</p>
```

Comment:
- [ ] Good job adding all the elements required :+1:, but I think it would be better if you change the `LET'S CONNECT` subtitle for the correct semantic tag `hx`.